domain/src/main/java/com/anytypeio/anytype/domain/page/AddBackLinkToObject.kt

```diff
@@ -23,7 +22,11 @@ class AddBackLinkToObject(
 
 
     override suspend fun doWork(params: Params): ObjectWrapper.Basic {
-        when (val result = openPage.run(params.objectToPlaceLink)) {
+        val openPageParams = OpenPage.Params(
+            obj = params.objectToPlaceLink,
+            saveAsLastOpened = true
```

Comment:
consider adding this option to params to able to change value

Reply:
done